# HG changeset patch
# User Rik <rik@octave.org>
# Date 1544370035 28800
#      Sun Dec 09 07:40:35 2018 -0800
# Node ID ef0998af70f9436cdd9cc3a0271dc0cba66ff786
# Parent  572329faeb3bcc417e642acceb931945b3985724
Change "Margin" default of text object to 3.

* NEWS: Announce change.

* graphics.in.h (text): Change Margin default to 3 from 2.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -93,16 +93,19 @@ Summary of important user-visible change
 
  ** "FaceNormals" and "VertexNormals" for patch and surface graphic
     objects are now calculated only when necessary to improve graphics
     performance.  In order for any normals to be calculated the
     "FaceLighting" property must be set to "flat" (FaceNormals) or
     "gouraud" (VertexNormals), AND a light object must be present in the
     axes.
 
+ ** The "Margin" property of text() objects has a new default of 3
+    rather than 2.  This change was made for Matlab compatibility.
+
  ** Printing to raster formats (bitmaps like PNG or JPEG) now uses an
     OpenGL-based method by default.  The print options "-opengl"
     (raster) and "-painters" (vector) have been added ("qt" toolkit
     only).  The figure property "renderer" specifies which renderer to
     use.  When the property "renderermode" is "auto" Octave will select
     -opengl for a raster output format and -painters for a vector output
     format.
 
diff --git a/libinterp/corefcn/graphics.in.h b/libinterp/corefcn/graphics.in.h
--- a/libinterp/corefcn/graphics.in.h
+++ b/libinterp/corefcn/graphics.in.h
@@ -4360,17 +4360,17 @@ public:
       string_property fontname u , OCTAVE_DEFAULT_FONTNAME
       double_property fontsize u , 10
       radio_property fontunits SU , "inches|centimeters|normalized|{points}|pixels"
       radio_property fontweight u , "light|{normal}|demi|bold"
       radio_property horizontalalignment mu , "{left}|center|right"
       radio_property interpreter u , "{tex}|none|latex"
       radio_property linestyle , "{-}|--|:|-.|none"
       double_property linewidth , 0.5
-      double_property margin , 2
+      double_property margin , 3
       array_property position smu , Matrix (1, 3, 0.0)
       double_property rotation mu , 0
       text_label_property string u , ""
       radio_property units u , "{data}|pixels|normalized|inches|centimeters|points"
       radio_property verticalalignment mu , "top|cap|{middle}|baseline|bottom"
 
       // hidden properties for limit computation
       row_vector_property xlim hlr , Matrix ()
